metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -131,6 +131,13 @@ def step_init(obj, run_id, step_name, passed_in_split_indexes, task_id):
     help="Generate the KFP YAML which is used to run the workflow on Kubeflow Pipelines.",
     show_default=True,
 )
+@click.option(
+    "--yaml-format",
+    "yaml_format",
+    default="kfp",
+    help="One of 'kfp', 'argo-workflow' or 'argo-workflow-template'.",
+    show_default=True,
```

Comment:
This could instead be https://click.palletsprojects.com/en/8.1.x/options/#choice-options

I asked the Edge browser :) 

<img width="360" alt="image" src="https://user-images.githubusercontent.com/4167032/231311153-1cad49f2-7f06-4197-a229-6dcd59aa664d.png">